.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -20,18 +20,42 @@ jobs:
         CONFIG: linux_64_numpy1.19python3.9.____cpython
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
+      linux_64_numpy1.21python3.10.____cpython:
+        CONFIG: linux_64_numpy1.21python3.10.____cpython
+        UPLOAD_PACKAGES: 'True'
+        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
       linux_aarch64_numpy1.18python3.7.____cpython:
         CONFIG: linux_aarch64_numpy1.18python3.7.____cpython
         UPLOAD_PACKAGES: 'True'
-        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-aarch64
+        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
```

Comment:
When re-rendering locally this appears to have changed, which doesn't seem right. Going to ask the bot to re-render as well

Reply:
cc @conda-forge/core

Reply:
Moving below to make this clearer ( https://github.com/conda-forge/numba-feedstock/pull/87#pullrequestreview-852587560 )